keycloak/realm.json

```diff
@@ -21,10 +21,8 @@
           "value": "student"
         }
       ],
-      "realmRoles": [
-        "student"
-      ],
       "clientRoles": {
+        "next-client": [ "student" ],
```

Comment:
What if I want to call teacher's API from mobile app?

Reply:
@akash073 vai, what should we do about that?